app/main.py

```diff
@@ -1 +1,24 @@
-# write your code here
+import os
+
+
+def move_file(command):
+    origin = command.split(" ")[1]  # source file_name
```

Comment:
```suggestion
    origin = command.split(" ")[1]
```
We believe that code should be self-descriptive
So you don't need to write comments